Makefile

```diff
@@ -152,6 +165,25 @@ endif
 	rm -f LICRES
 	@echo "  all OK"
 
+# Check licenses of go dependencies
+.PHONY: go-license-check
+go-license-check: $(GO_LICENSES_BIN)
+	@echo "Checking third-party licenses"
+	@"$(GO_LICENSES_BIN)" check ./lib/go/... --include_tests --disallowed_types=forbidden,permissive,reciprocal,restricted,unknown
+	@echo "License checks OK"
+
+# Save licenses of go dependencies
+.PHONY: go-license-save
+go-license-save: $(GO_LICENSES_BIN)
+	@echo "Saving third-party license files"
+	@rm -rf "$(OUTPUT)/third-party-licenses"
+	@"$(GO_LICENSES_BIN)" \
+		save ./lib/go/... \
```

Comment:
Should we add `--include_tests` for testing code?

Reply:
We don't need to retain the licenses for test-only dependencies.